gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/master.yml

```
@@ -124,4 +124,6 @@ databaseChangeLog:
   - include:
       - file: liquibase/changelogs/v3_17_0/schema.yml
   - include:
-      - file: liquibase/changelogs/v3_18_0/schema.yml
\ No newline at end of file
+      - file: liquibase/changelogs/v3_18_0/schema.yml
+  - include:
+      - file: liquibase/changelogs/v3_18_0/schema_flows.yml
```

If we target master then it will land on 3.19
```suggestion
      - file: liquibase/changelogs/v3_19_0/schema.yml 
```

---

yes i try to change the target to 3.18